packages/clerk-js/src/ui/utils/errorHandler.ts

```diff
@@ -15,10 +15,17 @@ function setFieldErrors(fieldStates: Array<FormControlState<string>>, errors: Cl
   }
 
   fieldStates.forEach(field => {
-    const error = errors.find(err => {
+    let errorCallback = field?.callback;
+
+    if (!errorCallback) {
+      errorCallback = (err: ClerkAPIError[]) => err[0];
+    }
+
+    const error = errors.filter(err => {
```

Comment:
🔧 The `error` variable here is an array of one or more errors. We should rename it.